site/css/mixins.scss

```diff
@@ -260,3 +270,43 @@
     box-shadow: 0 5px 15px rgba(0, 0, 0, 0.3);
     opacity: 0.9;
 }
+
+@mixin overlay {
+    position: fixed;
+    left: 0;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    background-color: rgba(0, 0, 0, 0.4);
+}
+
+@mixin mobile-toggle-icon {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 40px;
+    height: 40px;
+    padding: 0;
+    font-size: 1.125rem;
+}
+
+// Polyfills for flexbox gap (https://caniuse.com/flexbox-gap)
+// In the (near) future, this can be removed and replaced with (column-)gap: $gap
+@mixin column-gap($gap) {
```

Comment:
Thanks for the review @ikesau!

> One thing I noticed: I can navigate the categories with tab, but I can't nest into the topic area. It's such a fine menu, it would be a shame for it to not be keyboard accessible. 😊

Yes that bugged me too, let's see what we can do about it.

> Just wondering, when's the near future? Do we have an official coverage threshold?

I tend to go with 95% (it is currently around 92%, and keeping in mind these are not coming from our audience, with all the caveats it implies), although we might want to be even more conservative in this case since it affects layout.